Makefile

```diff
@@ -17,16 +17,20 @@ mypy: ## Run typeckecking according to mypy configuration in setup.cfg
 .PHONY: build_docs
 build_docs:
 	# readthedocs.org build command
-	python -m sphinx -T -b html -d _build/doctrees -D language=en doc/source  doc/_build/html
+	python -m sphinx -T -b html -d doc/_build/doctrees -D language=en doc/source  doc/_build/html
 
 .PHONY: build_latex
 build_latex: ## Build the documentation into a pdf
 	# readthedocs.org build command
 	# explicit cd here due to a bug in latexmk 4.41
-	python -m sphinx -b latex -d _build/doctrees -D language=en doc/source doc/_build/latex && \
+	python -m sphinx -b latex -d doc/_build/doctrees -D language=en doc/source doc/_build/latex && \
 	cd doc/_build/latex && \
 	latexmk -pdf -f -dvi- -ps- -jobname=alibi-detect -interaction=nonstopmode
 
+.PHONY: build_latex
+build_latex: ## Build the documentation into a pdf
+	$(MAKE) -C doc latexpdf
+
```

Comment:
This seem redundant (and overlaps the targets) given the above command.

Reply:
Oops must have screwed up that merge resolution. Will delete.